packages/flutter/lib/src/material/tabs.dart

```diff
@@ -570,6 +575,63 @@ class _IndicatorPainter extends CustomPainter {
     _painter!.paint(canvas, _currentRect!.topLeft, configuration);
   }
 
+  /// Applies the stretch effect to the indicator.
+  Rect _applyStretchEffect(Rect rect) {
+    final double index = controller.index.toDouble();
+    final double value = controller.animation!.value;
+
+    // The progress of the animation from 0 to 1
+    double tabChangeProgress;
```

Comment:
Should this be `late`?

Reply:
I guess it could, but I've always wondered. Doesn't the variable act as late without specifying the keyword in this situation? Because it is non null and without an initial value.
I don't know if it is syntactic sugar or if it has different behavior for this particular situation.

Reply:
It may be implied, I was actually surprised analysis did not fail.

Reply:
Yes, me too when I discovered it a few months ago. I don't think this was the case when `late` was introduced :smile: 
I've included the keyword anyway, it's more explicit.